CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+- Migrate publish flow from Jenkins to GitHub Actions
+- Add devops-tooling as owner for .github/workflows/cicd_*
```

Comment:
Is it necessary to include these details in public changelog?

If there even is a need to release a new version, I would keep this information more vague. If it can just be merged to master, then I would exclude adding changelog at all 🤔

Reply:
I updated the readme to be more vague. The patch label is needed to test that the release process is working.